attbillsplitter/main.py

```diff
@@ -141,16 +141,27 @@ def login(self):
             return True
         else:
             if 'promo' in login_submit.url.lower():
-                print ('\U0001F534  Popup window detected during login. '
+                reject_status = self.clickSkipPromo()
+
+                if reject_status == False:
+                    print ('\U0001F534  Popup window detected during login. '
                        'Please login you account in a browser and click '
                        'through. Log out your account before you retry. '
                        '(Sometimes you might have to do this multiple times.')
-                return False
+
+                return reject_status
 
             print('\U0001F6AB  Login failed. Please check your username and '
                   'password and retry. Or something unexpected happened.')
             return False
 
+    def clickSkipPromo(self):
```

Comment:
I've been using underscores over camelcase in this project. Would you mind changing it to `click_skip_promo`?